integration-tests/environments/react-native/package.json

```diff
@@ -31,16 +31,9 @@
     },
     "pod-install:simulator": {
```

Comment:
As we talked about on Zoom, this could be collapsed into a single "pod-install" now that it doesn't have platform specific "build" dependencies.